ios/RCTTWVideoModule.m

```diff
@@ -173,7 +174,11 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   if (self.camera == nil) {
       return;
   }
-  self.localVideoTrack = [TVILocalVideoTrack trackWithSource:self.camera enabled:NO name:@"camera"];
+  if (self.customVideoTrackName != nil) {
```

Comment:
We might instead update this to allow for a parameter rather than store the property since we're not natively handling a lifecycle that needs it.